server/index.js

```diff
@@ -45,8 +45,12 @@ app.get('/:listingID/averageReviewsRating', (req, res) => {
   db.getAverageReviewRating(req.params.listingID).then((ratings) => {
     let ratingsValue = Object.values(ratings[0]);
     ratingsValue.shift();
-    ratingsValue.forEach((rating) => rating.toFixed(1))
-    let averageRating = (ratingsValue.reduce((a,b) => a + b, 0 ))/(ratingsValue.length)
+    for(var key in ratings[0]) {
+      if(key !== '_id') {
+        ratings[0][key] = ratings[0][key].toFixed(1)
+      }
+    }
+    let averageRating = (ratingsValue.reduce((a,b) => a + b, 0 ))/(ratingsValue.length);
```

Comment:
hey @nelsonly123 feel like it would be redundant here to change all the average reviews to a string to add a decimal point only to convert them back to a number and average it all up. i think it would be easier for you to change the seeding script to included the 'precision' option in faker for numbers 

from your seeding script:

` reviewRating: {
          cleanliness: faker.random.number({min:1, max:5}),
          communication: faker.random.number({min:1, max:5}),
          checkIn: faker.random.number({min:1, max:5}),
          accuracy: faker.random.number({min:1, max:5}),
          location: faker.random.number({min:1, max:5}),
          value: faker.random.number({min:1, max:5}),
        },`

i believe you can do 
` reviewRating: {
          cleanliness: faker.random.number({min:1, max:5, precision: 0.1}),
          communication: faker.random.number({min:1, max:5, precision: 0.1}}),
          checkIn: faker.random.number({min:1, max:5, precision: 0.1}}),
          accuracy: faker.random.number({min:1, max:5, precision: 0.1}}),
          location: faker.random.number({min:1, max:5, precision: 0.1}}),
          value: faker.random.number({min:1, max:5, precision: 0.1}}),
        },`

I think this would be the easier approach and we can solve your issue of precision / decimal points prior to even coming into your server route. Thoughts?